README.md

```diff
@@ -152,6 +152,20 @@ spark-submit --properties-file cdm.properties \
 - When CDM migration (or validation with autocorrect) is run multiple times on the same table (for whatever reasons), it could lead to duplicate entries in `list` type columns. Note this is [due to a Cassandra/DSE bug](https://issues.apache.org/jira/browse/CASSANDRA-11368) and not a CDM issue. This issue can be addressed by enabling and setting a positive value for `spark.cdm.transform.custom.writetime.incrementBy` param. This param was specifically added to address this issue.
 - When you rerun job to resume from a previous run, the run metrics (read, write, skipped, etc.) captured in table `cdm_run_info` will be only for the current run. If the previous run was killed for some reasons, its run metrics may not have been saved. If the previous run did complete (not killed) but with errors, then you will have all run metrics from previous run as well.
 
+# Performance FAQ 
+- Below recommendations may only be needed while migrating large tables where the default performance is not good enough. 
+- Performance bottleneck are usually the result of
+    - Low resource availability on `Origin` OR `Target` cluster
+    - Low resource availability on CDM VMs, [see recommendations here](https://docs.datastax.com/en/data-migration/deployment-infrastructure.html#_machines)
+    - Bad schema design which could be cause by Out of balance `Origin` cluster, large partitions (> 100 MB), large rows (> 10MB) and/or high column count
+- Incorrect configuration of below properties
+    - `numParts`: Default is 5K, but ideal value is usually around table-size/10MB. 
+    - `batchSize`: Default is 5, but this should be set to 1 for tables where primary-key=partition-key OR where average row-size is > 20 KB. Similarly, this should be set to a value > 5, if row-size is small (< 1KB) and most partitions have several rows (100+).
+    - `fetchSizeInRows`: Default is 1K & this usually fine. However you can reduce this if your table has many large rows (over 100KB).
+    - `ratelimit`: Default is 20K. Once you set all the other properties appropriately, set this value to the highest possible value that your cluster (origin & target) is able to handle.
```

Comment:
I wonder why we left `ratelimit` in non-camelCase format 🤔

Reply:
It does sound like a single word, but TBH, no clue!